addOns/network/src/main/java/org/zaproxy/addon/network/internal/server/http/LocalServerConfig.java

```diff
@@ -111,6 +113,19 @@ public LocalServerConfig(LocalServerConfig other) {
         updateFrom(other);
     }
 
+    /**
+     * Constructs a {@code LocalServerConfig} from the given instance and with the given alias
+     * checker.
+     *
+     * @param other other instance with the data.
```

Comment:
```suggestion
     * @param other another instance with the data.
```
I don't recall if this was used elsewhere, but maybe a minor tweak so that it doesn't seem like a typo...